### PR TITLE
Add libspeexdsp build manifest

### PR DESCRIPTION
**Pull request by melis-m**
```
[!]      Some built files haven't been moved to any package:
[!]          usr/lib64/libspeexdsp.la
[+]      Wrapping sys-libs/libspeexdsp#1.2.0
[+]          name: libspeexdsp
[+]          category: sys-libs
[+]          version: 1.2.0
[+]          description: Speex is an Open Source/Free Software patent-free audio compression format designed for speech.
[+]          tags: sound, speech, libs, gnu
[+]          maintainer: matteo.melis@epiteh.eu
[+]          licenses: bsd
[+]          upstream_url: https://www.speex.org/
[+]          kind: effective
[+]          wrap_date: 2019-11-15T14:59:29Z
[+]          dependencies:
[+]              sys-libs/glibc
[+]         
[+]          Files added:
[+]              ./usr/lib64/libspeexdsp.so.1 -> libspeexdsp.so.1.5.1
[+]              ./usr/lib64/libspeexdsp.so.1.5.1
[+]              ./usr/lib64/pkgconfig/speexdsp.pc
[+]          (That's 3 files.)
[+]          Creating data.tar.gz
[+]          Creating manifest.toml
[+]          Creating libspeexdsp-1.2.0.nest
[+]      Wrapping sys-libs/libspeexdsp-dev#1.2.0
[+]          name: libspeexdsp-dev
[+]          category: sys-libs
[+]          version: 1.2.0
[+]          description: Headers and manuals to compile or write a software using the sys-libs/libspeexdsp package.
[+]          tags: sound, speech, libs, gnu
[+]          maintainer: matteo.melis@epiteh.eu
[+]          licenses: bsd
[+]          upstream_url: https://www.speex.org/
[+]          kind: effective
[+]          wrap_date: 2019-11-15T14:59:29Z
[+]          dependencies:
[+]              sys-libs/libspeexdsp#=1.2.0
[+]         
[+]          Files added:
[+]              ./usr/include/speex/speexdsp_types.h
[+]              ./usr/include/speex/speex_echo.h
[+]              ./usr/include/speex/speex_jitter.h
[+]              ./usr/include/speex/speex_preprocess.h
[+]              ./usr/include/speex/speex_resampler.h
[+]              ./usr/include/speex/speexdsp_config_types.h
[+]              ./usr/lib64/libspeexdsp.so -> libspeexdsp.so.1.5.1
[+]              ./usr/lib64/libspeexdsp.a
[+]          (That's 8 files.)
[+]          Creating data.tar.gz
[+]          Creating manifest.toml
[+]          Creating libspeexdsp-dev-1.2.0.nest
[+]      Wrapping sys-libs/libspeexdsp-doc#1.2.0
[+]          name: libspeexdsp-doc
[+]          category: sys-libs
[+]          version: 1.2.0
[+]          description: Offline documentation of sys-libs/libspeexdsp.
[+]          tags: sound, speech, libs, gnu
[+]          maintainer: matteo.melis@epiteh.eu
[+]          licenses: bsd
[+]          upstream_url: https://www.speex.org/
[+]          kind: effective
[+]          wrap_date: 2019-11-15T14:59:29Z
[+]          dependencies:
[+]         
[+]          Files added:
[+]              ./usr/share/doc/speexdsp/manual.pdf
[+]          (That's 1 files.)
[+]          Creating data.tar.gz
[+]          Creating manifest.toml
[+]          Creating libspeexdsp-doc-1.2.0.nest
[+]  Done!
```

### CODE REVIEW

File: sys-libs/libspeexdsp.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+from stdlib.manifest import manifest
+
+
+@manifest(
+    name='libspeexdsp',
+    category='sys-libs',
+    description='''
+    Speex is an Open Source/Free Software patent-free audio compression format designed for speech.
+    ''',
+    tags=['sound', 'speech', 'libs', 'gnu'],
```
Comment (Arignir) [inline sys-libs/libspeexdsp.py]:
```suggestion
    tags=['sound', 'speech', 'audio', 'media'],
```

**Review by Arignir — CHANGES_REQUESTED** [review]

File: sys-libs/libspeexdsp.py (hunk 1 above)
Comment (melis-m) [reply sys-libs/libspeexdsp.py]: Why remove the `gnu` tag ? [Speex](https://www.speex.org/) is part of the GNU project

Comment (Arignir) [reply sys-libs/libspeexdsp.py]: Couldn't find it in savanaah but looks like you're correct according to their home page.

Comment (melis-m) [reply sys-libs/libspeexdsp.py]: Also https://www.gnu.org/software/
(https://www.gnu.org/software/speex/ redirects to https://www.speex.org/)

**Review by Arignir — APPROVED** [review]

**Pull request merged**